2022/CVE-2022-20680.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
Expected location (2022/20xxx/CVE-2022-20680.json) of file does not match actual location (2022/CVE-2022-20680.json)